nbsite/cmd.py

```diff
@@ -101,7 +110,8 @@ def generate_rst(
         offset=0,
         overwrite=False,
         nblink='bottom',
-        skip=''):
+        skip='',
+        strip_numbers=False):
```

Comment:
Should be True by default, right?

Reply:
I don't think it should be true by default, seems like it could be rather confusing, but I don't have a strong opinion.

Reply:
I think it should be true by default in our PyViz projects; can that be achieved outside of nbsite? If so, sure; leave it False here and set it up just for PyViz.